pages/common/hg-clone.md

```diff
@@ -0,0 +1,23 @@
+# hg clone
+
+> Create a copy of an existing repository in a new directory.
+
+- Clone a repository to a specified directory:
+
+`hg clone {{remote_repository_source}} {{destination_path}}`
+
+- Clone a repository on a specific branch:
+
+`hg clone --branch {{branch}} {{remote_repository_source}}`
+
+- Clone a repository with a clean working directory:
+
+`hg clone --noupdate {{remote_repository_source}}`
+
+- Clone a repository to a specific revision, tag or branch:
```

Comment:
So the `--updaterev` flag takes revision and branch both ? Why should someone use the `--branch` flag then ? And not use `--updaterev` always.

Reply:
`--updaterev` will checkout the specified revision, tag or branch.

I'll mention that `--branch` only clones the specified branch. And ignores the others unless they are pulled.